src/index.json

```diff
@@ -284,9 +284,9 @@
         ],
         "webapp": [
             {
-                "filename": "webapp-0.2.10-py2.py3-none-any.whl",
-                "sha256Digest": "69f9781e886a55961cd1dd7d168045299ec8dbcfae4f9e17bc10ce3bfd1d52db",
-                "downloadUrl": "https://github.com/panchagnula/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.10-py2.py3-none-any.whl",
+                "filename": "webapp-0.2.11-py2.py3-none-any.whl",
+                "sha256Digest": "8cea6558656509d3f6d4c93c586502f5ad5f7d9420aa5265b83c501089169dec",
+                "downloadUrl": "https://github.com/Nking92/azure-cli-extensions/raw/sisirap-extensions-whl/dist/webapp-0.2.11-py2.py3-none-any.whl",
```

Comment:
Not blocking this PR, but ideally, the whl files would be hosted on a dedicated source. 
Azure Blob Storage, Pypi, dedicated github repo would all work.

Reply:
I'll discuss this with @panchagnula once she's back. I agree that this way of hosting the file is fragile and confusing.